modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/elasticsearch/ElasticsearchConfig.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.modelservice.elasticsearch;
+
+import com.expedia.adaptivealerting.modelservice.exception.MissingSystemPropertyException;
+import org.apache.http.HttpHost;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.elasticsearch.core.ElasticsearchRestTemplate;
+import org.springframework.data.elasticsearch.repository.config.EnableElasticsearchRepositories;
+
+/**
+ * Elastic search config used by spring data elastic search
+ */
+@Configuration
+@EnableElasticsearchRepositories(basePackages = "com.expedia.adaptivealerting.modelservice.repo")
+public class ElasticsearchConfig {
+
+    @Autowired
+    private ElasticSearchProperties properties;
+
+    @Bean
+    public ElasticsearchRestTemplate elasticsearchTemplate() {
+        return new ElasticsearchRestTemplate(client());
+    }
+
+    @Bean(destroyMethod = "close")
+    public RestHighLevelClient client() {
+        String url = properties.getUrls();
+        if (url == null) {
+            throw new MissingSystemPropertyException("Elastic search URL not set in config");
+        }
+
+        String[] arrOfUrl = url.split(":");
```

Comment:
yes. let me work on that.

Reply:
actually, for that, I will have to update the existing config. I will add a TODO comment and later change the config to have host and port instead of storing the whole URL as a string.

Reply:
done. I have left a TODO comment as well as we don't want to store URL as a string.